keps/sig-node/3673-kubelet-parallel-image-pull-limit/README.md

```diff
@@ -851,6 +851,20 @@ This through this both in small and large cases, again with respect to the
 
 No.
 
+###### Can enabling / using this feature result in resource exhaustion of some node resources (PIDs, sockets, inodes, etc.)?
+
+<!--
+Focus not just on happy cases, but primarily on more pathological cases
+(e.g. probes taking a minute instead of milliseconds, failed pods consuming resources, etc.).
+If any of the resources can be exhausted, how this is mitigated with the existing limits
+(e.g. pods per node) or new limits added by this KEP?
+
+Are there any tests that were run/should be run to understand performance characteristics better
+and validate the declared limits?
+-->
+
+No. 
```

Comment:
it can though. If configured improperly, customer may cause disk IO or network issues. I think the right answer here is to say "it can cause disk IO or network exhaustion, but it is not a new problem if parallel image pull is enabled. And this KEP is actually allowing to avoid this exhaustion"

Reply:
yep this KEP is only adding the limit of parallel image pulling, not enabling parallel image pulling itself. So enabling this feature when parallel pulling is enabled will only make things better. 

I will add more details here

Reply:
Done. PTAL